protos/frame-pointer-validator.proto

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+syntax = "proto3";
+
+message ModuleInformation {
+  string module_path = 1;
+  repeated FunctionInformation functions = 2;
```

Comment:
move these to request.

Reply:
But how would I do this? Both contain repeated fields.

Reply:
Let's do one request per module - let the client generate multiple requests if multiple modules need to be validated. This also makes error handling easier.

Reply:
The issue with this, is that I would then need to keep track of those in the UI and aggregate the results there.

Reply:
Nevermind, I changed it that way.